src/ArcGISProToArcGISEarth/HelpButton.cs

```diff
@@ -12,16 +12,28 @@
 // limitations under the License.
 
 using ArcGIS.Desktop.Framework.Contracts;
+using System;
 using System.Diagnostics;
 
-namespace ToArcGISEarth
+namespace ArcGISProToArcGISEarth
 {
     public class HelpButton : Button
     {
         protected override void OnClick()
         {
-            // Open the help page. 
-            Process.Start("http://doc.arcgis.com/en/arcgis-earth/automation-api/samples.htm");
+            try
+            {
+                // Open the help page. 
+                Process.Start(new ProcessStartInfo
+                {
+                    FileName = "http://doc.arcgis.com/en/arcgis-earth/automation-api/samples.htm",
```

Comment:
Replace the HTTP address with the HTTPS:
https://doc.arcgis.com/en/arcgis-earth/automation-api/samples.htm